### PR TITLE
chore: prettier, ci, community file cleanup

### PR DESCRIPTION
**Pull request by kanadgupta**
Meant to suggest this in #72 but was too slow 😭 just does a little bit more pre-release housekeeping:
- [x] Runs tests against more Node versions
- [x] Runs prettier on everything
- [x] Removes several community health files in favor of our [shared organization files](https://github.com/readmeio/.github/tree/main/.github)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x]
+        node-version: [14, 16, 18, 20]
```
Comment (dannobytes) [inline .github/workflows/ci.yml:11]: lol, nice. This is probably better in a separate PR anyway 💨

File: package.json [hunk 2]
```diff
@@ -10,7 +10,8 @@
     "prebuild": "rm -rf dist",
     "prepack": "npm run build",
     "pretest": "npm run lint && npm run prettier",
-    "prettier": "prettier --check --write \"./**/**.{js,ts,md,json}\"",
+    "prettier": "prettier --check .",
```
Comment (kanadgupta) [inline package.json:13]: I noticed that `prettier` is run via a `pretest` hook so we shouldn't be using the `write` flag here

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -18,9 +18,6 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
 
-      - name: Install npm@7
```
Comment (kanadgupta) [inline .github/workflows/ci.yml:21]: No longer required since we're testing against Node 16+

File: package.json [hunk 4]
```diff
@@ -10,7 +10,8 @@
...
+    "prettier:write": "prettier --check --write .",
```
Comment (dannobytes) [inline package.json:14]: I like the separation better too!

What do you think about adding one for `lint` too, but consolidating the name to `...:fix`?
```
"lint:fix": "eslint . --ext .js,.ts --fix",
...
"prettier:fix": "prettier --check --write .",
```

**Review by dannobytes — APPROVED** [review]
Just a suggestion to add "fix" scripts for lint too and maybe stay consistent with the name? Otherwise, great!

File: package.json (hunk 4 above)
Comment (kanadgupta) [reply package.json]: great call, added in https://github.com/readmeio/http-status-codes/pull/73/commits/3aed6cca056d49f5380cd688416d26b1ad273576!

**Review by erunion — APPROVED** [review]

**Pull request merged**